### ISSUE TITLE
Cura Update 3.2.1 removied printers and unable to add Ender Pro v2 back into software

### ISSUE THREAD

**Issue by Jafloonter** [Ultimaker/Cura#9808]
### Application Version

3.2.1

### Platform

Windows 10

### Printer

Printer no longer listed

### Reproduction steps

Install Cura version 3.2.1
Have existing printers wiped out
Try to add Creality Ender Pro v2, should be listed under printer name Creality 3D, yet this option is no longer available.

Suddenly unable to add printer back into Cura software that I have been using for years with this printer.

![No Ender Pro v2 Printer](https://user-images.githubusercontent.com/83999163/117757091-641e3980-b1ed-11eb-8a02-7ecc69326c6e.PNG)

### Actual results

Unable to add printer back into Cura software

### Expected results

An update should not reset or remove any printers

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Please resolve this issue or tell me how I can manually recreate this printer within the Cura software or whether I should be looking to use different software for my slicing needs.

**Comment by Asterchades** [comment]
To the best I can find, there is no "Ender Pro v2". There's an Ender 3 Pro, an Ender 5 Pro, and an Ender 3 v2, but nothing which combines both "pro" and "v2" designations.

More than that, all of the above printers are significantly newer than Cura 3.2.1, which was released in February 2018. The closest is the Ender 3 Pro, which was released in September 2018. As such it's not unexpected in the slightest that there won't be any profile for them.

All three of the above are, however, supported within versions of Cura newer than they are. Anything from 4.6 onward should have all three, while anything from 3.5 onward should have at least the Ender 3 Pro. And you can always add a Custom FFF/FDM printer to suit whatever platform you're planning to use, which often yields better results than the Creality profiles.

**Comment by Jafloonter** [comment]
This is interesting, I did not think to check that there was a newer version of Cura than the one that it prompted me to install.  I have not actually needed to worry about versions at all till now, it has always just simply worked perfect for my needs.  Previously  my printer was listed in the software under Creality 3D and from there the specific version.  It was very easy to set up which is why I found it so confusing that it both disappeared after my "upgrade" and was no longer an option.  

Based on your feedback, I found Cura v4.9 that I am downloading now so I do hope this resolves the issue.

My bug might be translated into why did Cura tell me there was a new version available and then installed 3.2.1?
